sdk/eventgrid/azure-eventgrid/azure/eventgrid/aio/_operations/_operations.py

```diff
@@ -132,11 +153,11 @@ async def _publish_cloud_events(  # pylint: disable=inconsistent-return-statemen
         _params = kwargs.pop("params", {}) or {}
 
         content_type: str = kwargs.pop('content_type', _headers.pop('content-type', "application/cloudevents-batch+json; charset=utf-8"))
-        cls: ClsType[None] = kwargs.pop(
+        cls: ClsType[_models._models.PublishResult] = kwargs.pop(  # pylint: disable=protected-access
             'cls', None
         )
 
-        _content = json.dumps(events, cls=AzureJSONEncoder)  # type: ignore
+        _content = json.dumps(events, cls=AzureJSONEncoder, exclude_readonly=True)  # type: ignore
```

Comment:
just double checking, has the default behavior always been to exclude readonly ?

Reply:
I believe so -- this was done by the code gen